src/extension/log/vscode-node/requestLogTree.ts

```diff
@@ -419,7 +419,28 @@ export class RequestLogTree extends Disposable implements IExtensionContribution
 				return;
 			}
 
-			const exportableItems = allTreeItems.filter(item => item instanceof ChatPromptItem || (item instanceof ChatRequestItem && item.info.entry.debugName !== 'modelList'));
+			// Whitelist of exportable debugName patterns
+			const isExportable = (debugName: string): boolean => {
+				// Include main conversation requests (location/intent format)
+				if (debugName.includes('/')) {
+					return true;
+				}
+
+				// Include specific API wrapper calls and BYOK providers
+				const exportableNames = [
```

Comment:
What's being filtered out? I'm unsure about hardcoding these names. What do we have to do when we add a new BYOK provider? Something called `debugName` is not usually a load-bearing feature